db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/ValueRow.java

```diff
@@ -68,6 +68,7 @@
 	private int ncols;
 	private int hash = 0; // Cached value of hashCode(). Invalidate on any change to the object.
 	private byte[] key;
+	private String[] names;
```

Comment:
I see that the column names stored in ValueRow is only used in SparkDataSet.writeORCFile(). If that is the case, to avoid the 4 bytes extra added to every ValueRow, why don't we directly pass the column names through the DWLWriteOperation?